include/ck/tensor_operation/gpu/device/device_batched_gemm_c_permutation.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <iostream>
+#include <vector>
+
+#include "device_base.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+
+template <typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CElementwiseOperation>
+struct DeviceBatchedGemmCPermutation : public BaseOperator
```

Comment:
the group GEMM version is called "Permute", here is "Permutation". Let's be consistent in naming